tempus_fugit_minecraft/shaders.py

```diff
@@ -0,0 +1,22 @@
+from pyglet.gl import *
+import pyglet.graphics as graphics
+import cmath as math
+from ctypes import *
+from tempus_fugit_minecraft.utilities import *
+
+class Shaders():
+    def __init__(self, model):
+        self.blockInformation = model._shown
+        '''Solves issue #7'''
+    def TurnOnEnvironmentLight(self):
+        glEnable(GL_LIGHTING)
+        glEnable(GL_LIGHT0)
+        lightpos = [0, -1, 0, 0.]
+        lightpos = (c_float * len(lightpos))(*lightpos)
+        glLightfv(GL_LIGHT0, GL_POSITION, lightpos)
```

Comment:
Sounds right. I completely forgot I could do it like that for some reason.

Reply:
I meant, we don't even need the separate function. It could be part of the constructor.

Reply:
I would rather keep it separated since it's going to be turning on and off the lights, as well as modify them.  It's mostly just to keep stuff separated and organized I think.